typescript/infra/config/environments/mainnet3/warp/configGetters/getBlastZeroNetworkUSDBWarpConfig.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  ChainMap,
+  IsmType,
+  RouterConfig,
+  TokenRouterConfig,
+  TokenType,
+} from '@hyperlane-xyz/sdk';
+
+import { safes } from '../../owners.js';
+
+const USDB_ADDRESS = '0x4300000000000000000000000000000000000003';
+
+export const getBlastZeroNetworkUSDBWarpConfig = async (
+  routerConfig: ChainMap<RouterConfig>,
+): Promise<ChainMap<TokenRouterConfig>> => {
+  const blast: TokenRouterConfig = {
+    ...routerConfig.blast,
+    type: TokenType.collateral,
+    token: USDB_ADDRESS,
+    interchainSecurityModule: {
+      owner: '0x723e7694dc346e5a15fB6F6A0144479aC624C66F',
```

Comment:
if we do need to keep it as a fallback routing hook, can we make sure that ownership of this is either set to the dead address, or have it be set to `routerConfig.<chain>.owner`?

Reply:
updated to address(0)